### PR TITLE
Updates to scripts to allow them to execute in Firefox without errors.

### PR DESCRIPTION
**Pull request by cillierscharl**
- Updated all scripts to allow them to be run in Firefox console (unable to run unless its wrapped in an async func)
- Fixed formatting 
- Fixed some random calc I scrolled down into wtf domi.

### CODE REVIEW

File: all_swaps.js [hunk 1]
```diff
@@ -1,105 +1,102 @@
...
+    for (let i in swapperBalance) {
+        if (swapperBalance[i] < -1) {
+            strPrint += "" + swapperTransactions[i][0].createdAt + " for $" + swapperTransactions[i][0].amount + " (" + swapperTransactions[i][0].direction + ") | " + i + " | " + swapperBalance[i].toFixed(2) + "\n";
+        }
     }
-}
-
-strPrint+="\n\nSo far you have swapped with "+tierCounter+" different Shakepay friends since April 20 🏓";
 
-swapperSinceMay3rdCounter=0;
```
Comment (cillierscharl) [inline all_swaps.js:98]: kekw

Comment (dlabrie) [reply all_swaps.js:98]: deal with it

File: auto_swap_initiator.js [hunk 2]
```diff
@@ -1,111 +1,112 @@
...
-        if(t.currency!="CAD") continue;
+    while (pullMore === true) {
+        console.log("pulling 2000 transactions from Shakepay api");
+        var transactionsResponse = await fetch("https://api.shakepay.com/transactions/history", { "headers": { "accept": "application/json", "accept-language": "en-US,en;q=0.9,fr;q=0.8", "authorization": window.sessionStorage.getItem("feathers-jwt"), "content-type": "application/json", "sec-ch-ua": "\" Not A;Brand\";v=\"99\", \"Chromium\";v=\"90\", \"Microsoft Edge\";v=\"90\"", "sec-ch-ua-mobile": "?0", "sec-fetch-dest": "empty", "sec-fetch-mode": "cors", "sec-fetch-site": "same-site" }, "referrerPolicy": "same-origin", "body": "{\"pagination\":{\"descending\":true,\"rowsPerPage\":2000,\"page\":" + page + "},\"filterParams\":{}}", "method": "POST", "mode": "cors", "credentials": "include" })
+        var transactionsData = await transactionsResponse.json();
 
-        createdAt = parseInt(Date.parse(t.createdAt));
-        if(createdAt < startTime.getTime()) {
+        var transactions = transactionsData.data;
+        if (transactions.length == 0) {
```
Comment (dlabrie) [inline auto_swap_initiator.js:34]: why did you do that? this is supposed to stop loading when it reaches transactions from the day before.

Comment (dlabrie) [reply auto_swap_initiator.js:34]: nevermind.

**Review by dlabrie — APPROVED** [review]

**Pull request merged**